docs/wallet-integration-guide/examples/scripts/exchange-integration/02-one-step-withdrawal.ts

```diff
@@ -0,0 +1,107 @@
+import { Label, TransferIn } from '@canton-network/core-ledger-client'
+import { pino } from 'pino'
+import { v4 } from 'uuid'
+import { setupExchange } from './setup-exchange.js'
+import { setupCustomer } from './setup-customer.js'
+import { tapParty } from './tap-party.js'
+import {
+    validateTransferIn,
+    validateTransferOut,
+} from './validate-transfers.js'
+
+const logger = pino({ name: '02-one-step-withdrawal', level: 'info' })
+
+const { exchangeParty, treasuryParty, treasuryKeyPair, exchangeSdk } =
+    await setupExchange()
+
+const { customerParty, customerKeyPair, customerSdk } =
+    await setupCustomer(true)
+
+const instrumentAdminPartyId =
+    (await exchangeSdk.tokenStandard?.getInstrumentAdmin()) || ''
```

Comment:
This feels off. This reads as if there is only one token supported?

If we go with this, then calling it `getCantonCoinAdminParty` would be better.